lib/taurus/qt/qtgui/editor/tauruseditor.py

```diff
@@ -123,27 +115,32 @@ def reload(self, idx=None, filename=None, goto=None):
     def set_current_filename(self, filename):
         self.editorStack().set_current_filename(filename)
 
+    def is_file_opened(self, filename=None):
+        # TODO: Implement it
+        return None
+
     def register_editorstack(self, editorstack):
         self.editorstacks.append(editorstack)
         if self.isAncestorOf(editorstack):
             # editorstack is a child of the Editor plugin
             editorstack.set_fullpath_sorting_enabled(True)
-            editorstack.set_closable(len(self.editorstacks) > 1)
+            editorstack.set_closable( len(self.editorstacks) > 1 )
             editorstack.set_outlineexplorer(self.outlineexplorer)
             editorstack.set_find_widget(self.find_widget)
             oe_btn = create_toolbutton(self)
             oe_btn.setDefaultAction(self.outlineexplorer.visibility_action)
             editorstack.add_corner_widgets_to_tabbar([5, oe_btn])
 
-        editorstack.set_io_actions(*self.io_actions)
+        if len(self.io_actions) == 0:
```

Comment:
Instead of this if, the createMenuActions method should return the dummy implementation (and document it in its docstring!)